kibbeh/src/ui/SettingsItemButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { BaseSettingsItem } from "./BaseSettingsItem";
+import { Button } from "./Button";
+
+export type SettingsItemButtonProps = {
+  text: string,
+  buttonText: string,
+  disabled?: boolean,
+  className: string,
+  onClick: (e: React.MouseEvent | React.TouchEvent) => void,
+};
+
+export const SettingsItemButton: React.FC<SettingsItemButtonProps> = ({
+  text,
+  buttonText,
+  disabled = false,
+  className = "",
+  onClick,
+}) => {
+  return (
+    <BaseSettingsItem className={`p-4 ${className}`}>
+      <p className="text-primary-200 mb-4">
```

Comment:
```suggestion
      <p className="text-primary-100 mb-4">
```

Reply:
> Can you pull latest? We've just merged the secondary bg for Button

Should I keep my change or was my change added already?